watchtower/wtclient/client_test.go

```diff
@@ -291,8 +291,8 @@ func (c *mockChannel) createRemoteCommitTx(t *testing.T) {
 
 	commitKeyRing := &lnwallet.CommitmentKeyRing{
 		RevocationKey: c.revPK,
-		NoDelayKey:    c.toLocalPK,
-		DelayKey:      c.toRemotePK,
+		RemoteKey:     c.toLocalPK,
```

Comment:
nice 👍 it case it isn't clear (from the lack of comments 😅) it's important to flip local and reverse before giving this to the wtclient since the logic is from the opposite perspective.